mibs/junos/ISIS-MIB

```diff
@@ -0,0 +1,4581 @@
+ISIS-MIB DEFINITIONS ::= BEGIN
```

Comment:
Hello
ISIS-MIB is a standard MIB, it must be located in the mibs folder, not in the ,mibs/junos folder. And in fact, is is already in the mibs folder, and seems to be the same version. Please check and remove the mibs/junos one.
Thanx

Reply:
For some reason the ISIS-polling does not seem to work at all when this file is removed. Do I need to specify the location for those generic MIBs?

Reply:
Found the issue - there seems to be an old MIB-file with possible incorrect OIDs in the junos-folder. Deleting that seems to resolve this issue.